server/moodle/mod/livequiz/mod_form.php

```diff
@@ -63,4 +67,25 @@ public function definition(): void {
 
         $this->add_action_buttons();
     }
+
+    /**
+     * Custom validation for the form.
+     *
+     * @param array $data  Submitted form data.
+     * @param array $files Uploaded files.
+     * @return array An array of validation errors, empty if none.
+     */
+    public function validation($data, $files): array {
+        return []; // No custom validation rules for now.
+    }
+
+    /**
+     * Preprocess form data before it is displayed.
+     *
+     * @param array &$default_values Default form values.
+     * @return void
+     */
+    public function data_preprocessing(&$defaultvalues): void {
+        // No preprocessing required for now.
```

Comment:
Ask team Lennert, we haven't touched this for anything but code sniffer compliance.
Not even sure it would show up in our PR if the skeleton PR hadn't been closed cus this pr exists

Reply:
This is your PR. I assume you have reasons for the things you want merged into the staging branch. If you do not know why something is in your code, perhaps it should not be in your code. If the reason is: the code is from a previous, now closed, PR, then it probably does not need to be here because a) that PR is now closed b) this is not that PR.